test/integration/epacems_test.py

```diff
@@ -75,3 +77,17 @@ def test_epacems_subset_input_validation(epacems_year_and_state, epacems_parquet
     for combo in combos:
         with pytest.raises(ValueError):
             epacems(epacems_path=path, **combo)
+
+
+def test_epacems_parallel(pudl_settings_fixture, pudl_ds_kwargs):
```

Comment:
I think this probably isn't the place to do this but there could be a speed test to make sure that the parallel version is working faster than the single process version.

Reply:
I think it's hard to get a representative sense of the speedup without processing a lot more data. I think the speedup will be bigger on big states (like FL, TX, CA, NY) since (I think) the overhead is mostly fixed per-process, and those big states sometimes have 100x as much data as the little ones like ID or VT. And generally we don't want to process *too* much data in CI. On my laptop I got about a 2x speedup going from 1 thread on 1 core to 4 threads on 2 cores. I'll be curious to see how that scales up with more cores/threads. Like if we run it on a 32 core node, it should take 5 minutes since every year gets its own CPU... but maybe it doesn't work out that cleanly.